docs/source/index.rst

```diff
@@ -10,20 +10,14 @@ Snowpark simplifies the process of building complex data pipelines and allows yo
 Snowflake directly without moving data to the system where your application code runs. For more
 information, see the `Snowpark Developer Guide for Python <https://docs.snowflake.com/en/developer-guide/snowpark/python/index.html>`_.
 
-.. rubric:: Modules
-
-.. autosummary::
-   :toctree: _autosummary
-   :template: module.rst
-
-   snowflake.snowpark
-   snowflake.snowpark.functions
-   snowflake.snowpark.table_function
-   snowflake.snowpark.stored_procedure
-   snowflake.snowpark.types
-   snowflake.snowpark.udf
-   snowflake.snowpark.udtf
-   snowflake.snowpark.exceptions
-
-:ref:`genindex`
```

Comment:
:ref:`genindex` renders as a [link to the index](https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/genindex.html). It's pretty handy so I hope we can keep it. Here's a screenshot:
![image](https://user-images.githubusercontent.com/85350026/197074015-cef824e4-0927-4050-a4c1-9bd301567909.png)

Reply:
It can go somewhere else if you prefer.